fearless/Modules/BackupSelectWallet/BackupSelectWalletInteractor.swift

```diff
@@ -24,7 +24,7 @@ extension BackupSelectWalletInteractor: BackupSelectWalletInteractorInput {
             do {
                 if let cloudStorageService = cloudStorageService {
                     cloudStorageService.disconnect()
-                    let accounts = try await cloudStorageService.getFearlessBackupAccounts()
+                    let accounts = try await cloudStorageService.getBackupAccounts()
```

Comment:
please make sure what `getBackupAccounts()` it's right method

Reply:
let's test